templates/myaccount/my-orders.php

```diff
@@ -31,7 +31,7 @@
 			<?php foreach ( $orders['orders'] as $order ) : ?>
 				<tr class="llms-order-item <?php echo $order->get( 'status' ); ?>" id="llms-order-<?php $order->get( 'id' ); ?>">
 					<td data-label="<?php _e( 'Order', 'lifterlms' ); ?>: ">
-						<a href="<?php echo $order->get_view_link(); ?>">#<?php echo $order->get( 'id' ); ?></a>
+						<a href="<?php echo esc_url( home_url( "/members/admin/courses/orders/{$order->get('id')}/" ) ); ?>">#<?php echo $order->get( 'id' ); ?></a>
```

Comment:
mmm but this way this link will only work on the buddyboss/buddypress profile page.
So here we're replacing a problem with another :D
I think this commit doesn't have so much sense and should be reverted, but feel free to tell me I'm wrong, I might have missed something...

Reply:
What do you suggest I should use?

By using esc_url() like this `esc_url( $order->get( 'id' )` it doesn't work and converts the ID into unexpected output.

Reply:
I'm reviewing the code and finding it doesn't fix the issue 100%.
I don't have a ready solution at the moment, I should spend some time on it, otherwise I would have already done a PR :P 

If you want to spend some more time on it it's fine, but not that much (a couple of hours?).
If you don't come up with a solution **it's fine**, you can add a comment to the issue with your findings etc, and remove the assignation to yourself, so that someone else would pick it up. <3

Reply:
@eri-trabiccolo 

> I'm reviewing the code and finding it doesn't fix the issue 100%.

What issues are you referring to?